packages/replay-worker/package.json

```diff
@@ -14,7 +14,7 @@
     "build:dev": "yarn build",
     "build:watch": "run-p build:transpile:watch build:types:watch",
     "build:dev:watch": "run-p build:watch",
-    "build:transpile:watch": "yarn build:rollup --watch",
```

Comment:
What is this about?

Reply:
just noticed this while working here that this is actually referencing wrong commands, so fixed them while at it! same for integration-shims, copy paste error 😅